node/src/service.rs

```diff
@@ -337,16 +355,19 @@ pub fn new_full(config: Configuration) -> Result<TaskManager, ServiceError> {
 
         let grandpa_config = sc_consensus_grandpa::Config {
             // FIXME #1578 make this available through chainspec
-            gossip_duration: Duration::from_millis(333),
+            gossip_duration: Duration::from_millis(1000),
```

Comment:
Is this affecting block finalization time as well ?

Reply:
It could. With this modification one GRANDPA round is potentially longer than before, and so finalization might be delayed accordingly.
Notice, we are talking about few seconds in the worst case anyway. For example, here is some interesting data on experimenting with a `2s` timeout versus `1s`.